meshtastic/mesh_interface.py

```
@@ -3093,243 +3502,299 @@ def _handle_queue_status_from_radio(
+    def _dispatch_from_radio_message(
+        self, context: _FromRadioContext
+    ) -> list[_PublicationIntent]:
+        """Dispatch normalized FromRadio payloads to dedicated branch handlers."""
+        branch = self._select_from_radio_branch(context)
+        if branch is None:
+            logger.debug("Unexpected FromRadio payload")
+            return []
+        handler = self._from_radio_dispatch_map()[branch]
+        return handler(context)
+
+    def _select_from_radio_branch(self, context: _FromRadioContext) -> str | None:
```

Ruff (0.15.6)

[warning] Too many return statements (14 > 6)

(PLR0911)